stylesheets/_touch-friendly-links.scss

```diff
@@ -0,0 +1,10 @@
+// This mixin will expand the clickable area for a link on mobile and tablet to make it easier to click on a touch
+// device.
+
+@mixin touch-friendly-links($padding: 3px) {
+  a {
+    padding: $padding;
+    margin: -1 * $padding;
+    outline: none;
```

Comment:
The easiest way to test is in Firefox – https://support.mozilla.org/t5/Manage-preferences-and-add-ons/Change-the-fonts-and-colors-websites-use/ta-p/579